IdentityServer/v5/docs/themes/hugo-theme-learn/layouts/partials/header.html

```diff
@@ -89,7 +89,7 @@
                 {{ end }}
               {{ end }}
               {{$toc := (and (not .Params.disableToc) (not .Params.chapter))}}
-              <div id="breadcrumbs" itemscope="" itemtype="http://schema.org/BreadcrumbList">
+              <div id="breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
```

Comment:
```suggestion
              <div id="breadcrumbs" itemscope="" itemtype="http://schema.org/BreadcrumbList">
```

Google insists on http